internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/fastclient/utils/AbstractClientEndToEndSetup.java

```diff
@@ -238,6 +239,8 @@ protected void prepareData() throws Exception {
     VersionCreationResponse creationResponse = veniceCluster.getNewStoreVersion(KEY_SCHEMA_STR, VALUE_SCHEMA_STR);
     storeVersionName = creationResponse.getKafkaTopic();
     storeName = Version.parseStoreFromKafkaTopicName(storeVersionName);
+    veniceCluster.useControllerClient(
+        client -> client.updateStore(storeName, new UpdateStoreQueryParams().setStorageNodeReadQuotaEnabled(true)));
```

Comment:
Instead of having this here, let's put this test in `TestParentControllerWithMultiDataCenter.testUpdateStore`?

Reply:
I can add a test in testUpdateStore but we also need to enable it here because FC tests should be executed with SN read quota enforced and there is a test `FastClientServerReadQuotaTest` that explicitly test SN read quota.

Reply:
Oh, I see. Then there is no need to add it in the other place